lib/open-amp/CMakeLists.txt

```
@@ -6,3 +6,6 @@
 
 zephyr_include_directories_ifdef(CONFIG_OPENAMP_RSC_TABLE .)
 zephyr_sources_ifdef(CONFIG_OPENAMP_RSC_TABLE resource_table.c)
+
+zephyr_compile_definitions_ifdef(CONFIG_OPENAMP_WITH_DCACHE WITH_DCACHE_VRINGS)
+zephyr_compile_definitions_ifdef(CONFIG_OPENAMP_WITH_DCACHE WITH_DCACHE_BUFFERS)
```

I think this simplification is possible:

```suggestion
zephyr_compile_definitions_ifdef(CONFIG_OPENAMP_WITH_DCACHE WITH_DCACHE_VRINGS WITH_DCACHE_BUFFERS)
```

---

yuuuup